server/src/controllers/note.controllers.ts

```diff
@@ -16,17 +29,45 @@ const buildJsonResponse = (data: unknown, key: string) => {
 
 export const deleteNote = async (req: Request, res: Response) => {
   const { noteId } = req.params;
-  const isDeleted = await Note.deleteNoteFromId(noteId);
-  if (isDeleted) {
-    res.status(200).send(`note ${noteId} was succesfully deleted`);
-  } else {
-    res.status(400).send(`note ${noteId} does not exist.`);
+  try {
+    const isDeleted = await Note.deleteNoteFromId(noteId);
+    if (isDeleted) {
+      res.status(200).send(`note ${noteId} was succesfully deleted`);
+    } else {
+      res.status(400).send(`note ${noteId} does not exist.`);
```

Comment:
This should be a `404` error. Check the list of error codes and their signification here: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#client_error_responses